hack/jenkins/e2e.sh

```diff
@@ -254,6 +409,38 @@ case ${JOB_NAME} in
     : ${PROJECT:="k8s-jkns-e2e-gce-release"}
     ;;
 
+  # kubernetes-upgrade-gke-1.0-release
+  #
+  # Configurations for step2, step4, and step6 live in the current release
+  # branch.
+
+  kubernetes-upgrade-gke-1.0-release-step1-deploy)
```

Comment:
Er, that was unclear.  We do *not* expect these jobs always to upgrade to 1.1; we only care about upgrading from 1.0 to current release.